README.md

```diff
@@ -19,191 +14,233 @@ pip install promote
 
-#### Usage
+**Example**
+```shell
+promote
+pandas
+pyodbc==4.0.24
+```
+<hr>
 
-`Promote.predict(model, data)`
+### `promote.sh`
+The `promote.sh` file can be included in your model directory. It is executed before your model is built and can be used to install low-level system packages such as Linux packages and other dependencies.
+
+**Example**
+```shell
+# Install Microsoft SQL Server RHEL7 ODBC Driver
+curl https://packages.microsoft.com/config/rhel/7/prod.repo > /etc/yum.repos.d/mssql-release.repo
+
+exit
+yum remove unixODBC-utf16 unixODBC-utf16-devel #to avoid conflicts
+ACCEPT_EULA=Y yum install msodbcsql17
+# optional: for bcp and sqlcmd
+ACCEPT_EULA=Y yum install mssql-tools
+echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bash_profile
+echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bashrc
+source ~/.bashrc
+```
+<hr>
 
-#### Arguments
-- `model`(_string_): the name of your model
-- `data`: data required to make a single prediction. This can be a dict or dataframe
+### <a name="helpers"></a>`helpers` Directory
+Users can also add a `helpers` directory to the root directory of their project to store helper scripts that are used by the deployment script. Please refer to the [Model Directory Structure](#directory) above for an example. Adding an `__init__.py` file to the `helpers` directory will allow the python files in the directory to be discoverable via the python `import` command.
 
-#### Example
+**Example**
 ```python
-p.predict("LPOptimizer_model", {"activities": ["sleep", "work", "leisure"], "required_hours": [7, 10, 0], "happiness_per_hour": [1.5, 1, 2]})
+from helpers import helper_funs
 ```
 <hr>
 
-### `Promote.metadata()`
-
-Store custom metadata about a model as part of the `PromoteModel()` when it is sent to the Promote servers. (limited to 6 key-value pairs)
-
-#### Usage
-
-`Promote.metadata[name] = value`
-
-#### Arguments
-- `name`(_string_): the name of your metadata (limit 20 characters)
-- `value`: a value for your metadata (limit 50 characters)
+### <a name="objects"></a>`objects` Directory
+Users can also add an `objects` directory to the root directory of their project to store helper scripts that are used by the deployment script. Please refer to the [Model Directory Structure](#directory) above for an example. The `objects` directory is a great place to put pretrained models and other model dependencies. It is a best practice to train models outside of the `deploy.py` script and to save the trained model to the `objects` directory. This prevents the Promote app from attempting to retrain the model on each redeploy.
 
-#### Examples
+**Example**
+```python
+my_model = pickle.load( open( "./objects/my_model.p", "rb" ) )
+```
+<hr>
 
+### Deployment
+Currently, the only way to deploy a python model is to execute the `deploy.py` script from a command line terminal. To do this, open a command line window, navigate to the root project directory and run the following
```

Comment:
Since I was trying to copy the R docs to an extent, that was language I inserted as copmarison. Since with R you can deploy from an interactive r studio session or command line, but from Python jupyter is not supported, that was the language I used. Happy to remove it though

Reply:
OK makes sense I will leave it as is